elasticsearch/linux_distributions/opendistro-tar-build.sh

```diff
@@ -85,6 +85,13 @@ for d in $PLUGINS_CHECKS; do
 done
 echo "Results: validated that plugins has been installed"
 
+# Download Knn lib 
+wget https://d3g5vo6xdbdb9a.cloudfront.net/downloads/k-NN-lib/libKNNIndexV1_7_3_6.zip \
+&& unzip libKNNIndexV1_7_3_6.zip \
+&& mkdir -p $PACKAGE_NAME-$OD_VERSION/plugins/opendistro-knn/knn-lib/ \
+&& mv libKNNIndexV1_7_3_6.so $PACKAGE_NAME-$OD_VERSION/plugins/opendistro-knn/knn-lib/ \
```

Comment:
Just to confirm, the expectation is to publish k-NN-lib/libKNNIndexV1_7_3_6.zip on a per version basis? If so, we should discuss a better way for versioning, such as packaging zip as opendistro-knnlib-X.X.X.X.zip.

Reply:
This PR is used to solve internet connectivity issue. These contents are just copied from the install script: https://github.com/opendistro-for-elasticsearch/opendistro-build/blob/master/elasticsearch/linux_distributions/opendistro-tar-install.sh#L49-L52 

I got your point. probably we need another PR to use the new lib zip file with right versioning. Let's have a discussion for that

Reply:
Sure, another PR would be good. I think from Plugin's side, we would just need to add to github action to zip libKNNIndexV1_7_3_6.so as opendistro-knnlib-X.X.X.X.zip instead. And then once that is changed, we could change here.